package.json

```diff
@@ -9,7 +9,6 @@
     "lint": "tslint -c tslint.json 'src/richie-front/js/**/*.ts?(x)'",
     "prettier": "prettier --write 'src/richie-front/js/**/*.+(ts|tsx|json|js|jsx)' '*.+(ts|tsx|json|js|jsx)'",
     "sass": "node-sass src/richie-front/scss/_main.scss src/richie/static/richie/css/main.css",
-    "test": "karma start ./karma.conf.js",
```

Comment:
Can we keep this test script `"test": "jest",`? And thus, use it in the `Makefile` and the CI as it was before this switch?

Reply:
I can see how it can be nice to keep the standard command throughout. Done.